### PR TITLE
Vue/vout

### PR DESCRIPTION
**Pull request by fanglinfang**

### CODE REVIEW

File: myuw/static/vue/components/_common/husky-experience.vue [hunk 1]
```diff
@@ -16,9 +16,11 @@
         >
           <img :srcset="srcset" :src="src" class="img-fluid" :alt="alt">
           <div class="position-absolute h-100 w-100 myuw-huskyexp-body">
-            <a :href="expLink"
-               :label="`${articleTeaserTitle}. ${articleTeaserBody}`"
-               class="d-block h-100 px-3 py-4"
+            <a
+              v-out="'articleTeaserTitle'"
```
Comment (SarthakSingh31) [inline myuw/static/vue/components/_common/husky-experience.vue:20]: Should this be `v-out="articleTeaserTitle"`?

File: myuw/static/vue/components/_templates/boilerplate/boilerplate.vue [hunk 2]
```diff
@@ -294,30 +296,30 @@
...
           title="MyUW video tour for staff"
-          data-linklabel="MyUW video for staff"
           >for staff</a
         >, or
         <a
           href="https://www.youtube.com/watch?v=K7GoUc32TMs&amp;t=5s&amp;list=PL-hNmjMg7KSHFdXj6yXDjZtCpjkkKBLUZ&amp;index=1"
           target="_blank"
           title="MyUW video tour for students"
-          data-linklabel="MyUW video for students"
+          dv-out="'MyUW video for students'"
```
Comment (SarthakSingh31) [inline myuw/static/vue/components/_templates/boilerplate/boilerplate.vue:316]: typo `dv-out`

File: myuw/static/vue/components/home/inst_course_summary/summary.vue [hunk 3]
```diff
@@ -6,7 +6,7 @@
   >
     <template #card-heading>
       <h3 class="h4 text-dark-beige myuw-font-encode-sans">
-        {{ titleCaseWord(getQuarter()) }} {{ getYear() }} Teaching Schedule
+        {{ titleCaseWord(getQuarter) }} {{ getYear }} Teaching Schedule
```
Comment (SarthakSingh31) [inline myuw/static/vue/components/home/inst_course_summary/summary.vue:9]: Why were the functions calls removed?

File: myuw/static/vue/components/_templates/boilerplate/boilerplate.vue (hunk 2 above)
Comment (fanglinfang) [reply myuw/static/vue/components/_templates/boilerplate/boilerplate.vue:316]: Thanks!

File: myuw/static/vue/components/_common/husky-experience.vue (hunk 1 above)
Comment (fanglinfang) [reply myuw/static/vue/components/_common/husky-experience.vue:20]: Yes, thanks!

File: myuw/static/vue/components/home/inst_course_summary/summary.vue (hunk 3 above)
Comment (fanglinfang) [reply myuw/static/vue/components/home/inst_course_summary/summary.vue:9]: I moved them from methods to computed.

File: myuw/static/vue/components/home/inst_course_summary/summary.vue [hunk 4]
```diff
@@ -32,25 +32,25 @@
...
           </p>
           <span>
-            <a
+            <b-link
+              v-out="'MyUW Teaching page'"
               :href="`/teaching/${term}`"
               :future-nav-target="`${term}`"
-              :data-linklabel="getTeachingLinkLabel()"
-              title="Open teaching page"
+              :title="getTeachingLinkLabel"
```
[the remark this answers is not in the document]
Comment (fanglinfang) [reply myuw/static/vue/components/home/inst_course_summary/summary.vue:39]: getTeachingLinkLabel is a computed variable

File: myuw/static/vue/components/_common/course/inst/linked-section.vue [hunk 5]
```diff
@@ -63,18 +65,18 @@
 
     <div>
       <b-button v-if="!section.mini_card"
-              variant="light"
-              :label="`Pin ${section.id} mini-card to teaching page`"
-              title="Click to pin the mini-card onto Teaching page"
-              @click="miniCard"
+        v-out="`Pin mini-card ${section.section_label}`"
```
Comment (fanglinfang) [inline myuw/static/vue/components/_common/course/inst/linked-section.vue:68]: I guess we don't need v-out here

File: myuw/static/vue/components/home/applicant/seattle.vue [hunk 6]
```diff
@@ -87,24 +89,24 @@
       </h5>
       <ul class="list-unstyled myuw-text-md">
         <li v-if="applicantData.is_freshman">
-          <a href="https://admit.washington.edu/apply/dates-deadlines/"
-             target="_blank" label="Key dates &amp; deadlines for freshmen"
+          <a v-out="'Key dates deadlines for freshmen'"
```
Comment (SarthakSingh31) [inline myuw/static/vue/components/home/applicant/seattle.vue:92]: Why do we provide a v-out value here? we can use the innerText

File: myuw/static/vue/components/accounts/tuition-fees.vue [hunk 7]
```diff
@@ -96,7 +96,9 @@
             <template #status-content>
               <div class="myuw-text-md">PCE-Continuum College</div>
               <div class="text-right">
-                <uw-link-button href="http://portal.continuum.uw.edu" target="_blank">
+                <uw-link-button v-out="'Make Continuum College tuition payment'"
```
Comment (fanglinfang) [inline myuw/static/vue/components/accounts/tuition-fees.vue:99]: Would v-out work on uw-link-button ?

File: myuw/static/vue/components/home/applicant/seattle.vue (hunk 6 above)
Comment (fanglinfang) [reply myuw/static/vue/components/home/applicant/seattle.vue:92]: Thanks!

**Review by SarthakSingh31 — APPROVED** [review]

**Pull request merged**